test/transpile/basic.spec.ts

```diff
@@ -0,0 +1,18 @@
+import * as path from "path";
```

Comment:
I'm kind of missing the point of that test, don't we already test all of that stuff in other tests? Also IMO it tests too much things

Reply:
I want to test this since this is the main use-case of the tool, even if parts of this are already tested, I don't think there is any other test that actually transpiles a project from disk (except bundle is separate since it has a different project setup).

Reply:
We have few tests with `tsconfig.json` in CLI, but yeah, we don't have any tests using `transpileProject` API. Maybe at least we may reduce scope of that test? I.e. rename it to `project` and remove that imaginary host API (which we already test)?